rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -474,7 +477,7 @@ public void release_verify_fee_above_fee_and_minimum_is_accepted() throws IOExce
     }
 
     @Test
-    public void processPegoutsInBatch_before_rskip_271() throws IOException {
+    public void test_processPegoutsInBatch_before_rskip_271_processPegoutsIndividually() throws IOException {
```

Comment:
You can remove this test, this case will be covered by `test_processPegoutsIndividually_before_RSKIP271_activation`

Reply:
Removed